ui/pages/destinations/[id]/index.js

```diff
@@ -120,7 +120,7 @@ export default function DestinationDetail() {
       </Head>
       <header className='mt-6 mb-12 space-y-4'>
         <div className=' flex flex-col justify-between md:flex-row md:items-center'>
-          <h1 className='flex truncate py-1 font-display text-xl font-medium'>
```

Comment:
Why 75%? Is the idea to add a buffer on the right to truncate?

Reply:
yup that is the reason why we add 75% there